internal/pkg/cli/pipeline_deploy_test.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/aws/copilot-cli/internal/pkg/config"
 	"github.com/aws/copilot-cli/internal/pkg/deploy"
 	deploycfn "github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation"
+	deploymock "github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation/mocks"
```

Comment:
Do we have to use the mocks from the other packages 👀  Could we declare the interface as well as the mock in `cli` package?

Reply:
I don't know what's going on with this package; for whatever reason my local mainline wouldn't build because of some interface issues. I'd prefer to remove this commit (and these pipeline changes) entirely. Probably will happen when I resolve merge conflicts.

Reply:
fixed by #5077; i'd like to get that merged to mainline before this PR